oscar_elasticsearch/search/format.py

```diff
@@ -120,3 +120,13 @@ def decimal3(key):
     return decimal.Decimal(key).quantize(
         decimal.Decimal("0.001"), decimal.ROUND_HALF_UP
     )
+
+
+def year_and_translated_month(value):
```

Comment:
this should not be here, or it should default be formatted to the Django default date format